app/application/adapter.js

```diff
@@ -47,7 +51,7 @@ export default DS.RESTAdapter.extend(DataAdapterMixin, {
    */
   decoratePayload(key, payload) {
     if (Array.isArray(payload[key])) {
-      payload[key].map(o => this.insertLink(key, o));
+      payload[key].map((o) => this.insertLink(key, o));
```

Comment:
is there a way to turn off this rule so it's more consistent with other files?
```suggestion
      payload[key].map(o => this.insertLink(key, o));
```

Reply:
set 'arrow-parens' rule to 'as-needed' https://eslint.org/docs/rules/arrow-parens#as-needed

Reply:
^yes that one!

Reply:
It actually was set in the prettier: `https://prettier.io/docs/en/options.html#arrow-function-parentheses`, let me add to make sure it is turned off.